Dockerfile

```diff
@@ -17,7 +17,4 @@ RUN poetry install --no-dev
 
 EXPOSE 5000
 
-ENV FLASK_APP=api.py
-
-ENTRYPOINT flask run --host 0.0.0.0
-
+CMD ["gunicorn", "api:app", "-b", "0.0.0.0:5000", "--workers", "20", "--worker-class", "uvicorn.workers.UvicornWorker", "--timeout", "0"]
```

Comment:
🎨 Very minor/cosmetic - Could maybe add line breaks for each 4 flags & values to help readability?

Reply:
Done this plus expanded on the "-b" attribute as well to make it more readable.